### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
Added Azure Monitor entry

### CODE REVIEW

**Review by pakrym — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -72,6 +72,9 @@
 # PRLabel: %Cognitive - Form Recognizer
 /sdk/formrecognizer/                            @annelo-msft @kinelski @maririos
 
+# PRLabel: %Azure Monitor
```
Comment (pakrym) [inline .github/CODEOWNERS:75]: There is no "Azure Monitor" label in the repo

Comment (pakrym) [reply .github/CODEOWNERS:75]: There are a few others though
image

Comment (AlexGhiondea) [reply .github/CODEOWNERS]: @rajkumar-rangaraj do we need to create a new label or any of the existing ones makes sense?

Comment (rajkumar-rangaraj) [reply .github/CODEOWNERS]: @AlexGhiondea It's better if we could create a new label with name `Monitor - Exporter`

Comment (rajkumar-rangaraj) [reply .github/CODEOWNERS]: Shall I create a new label and refer that in CODEOWNERS?

**Pull request merged**